py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -115,10 +146,9 @@ def batch_files(
         # use sys.getsizeof on batch event to get general size of payload
         # limit estimated payload size to 245KB for now to account for
         # additional request overhead and measurement inaccuracy
-        if (
-            batch.total_size + int(size) > threshold
-            or batch.event_size_over_limit()
```

Comment:
i took out over size limit warning since we shouldn't come close to this anymore?